Code/GraphMol/SynthonSpaceSearch/SearchResults.h

```diff
@@ -11,19 +11,25 @@
 #ifndef RDKIT_SYNTHONSPACE_SEARCHRESULTS_H
 #define RDKIT_SYNTHONSPACE_SEARCHRESULTS_H
 
+#include <functional>
 #include <RDGeneral/export.h>
 #include <GraphMol/ROMol.h>
 
 namespace RDKit::SynthonSpaceSearch {
 
+// takes vector of search results; returns true if enough hits have been
+// returned, false if the search should continue.
```

Comment:
Is this correct?  I don't see the return value being used in the ::search() function.